app/bills/page.tsx

```diff
@@ -48,74 +47,101 @@ export default async function BillSearchPage({
     searchParams.state
   );
 
+  console.log("bills", bills);
+
   const query = searchParams.query;
-  const state = searchParams.year;
-  const year = searchParams.state;
+  const state = searchParams.state;
+  const year = searchParams.year;
 
```

Comment:
Take a look at [destructuring assignment syntax](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#description). It's basically syntactic sugar for what you are doing here. Just a stylistic thing. I don't have a preference.